tangelo/linq/translator/translate_qubitop.py

```diff
@@ -35,7 +47,9 @@ def translate_operator(qubit_operator, source, target, n_qubits=None):
         qubit_operator (source format): Self-explanatory.
         source (string): Identifier for the source format.
         target (string): Identifier for the target format.
-        n_qubits (int): Number of qubits relevant to the operator.
+        n_qubits (int): Number of qubits relevant to the operator. It is
```

Comment:
We need to stop formulating comments that are so specific that they won't be true anymore every time code changes somewhere. We could instead say:

`Number of qubits relevant to the operator. Not used by all format conversion functions, will be computed automatically if not specified.`